pkg/admission/validator/shoot.go

```diff
@@ -111,8 +116,8 @@ func (s *shoot) validateContext(valContext *validationContext) field.ErrorList {
 	allErrors = append(allErrors, validation.ValidateInfrastructureConfig(valContext.infrastructureConfig, infrastructureConfigPath)...)
 	allErrors = append(allErrors, validation.ValidateControlPlaneConfig(valContext.controlPlaneConfig, controlPlaneConfigPath)...)
 	allErrors = append(allErrors, validation.ValidateWorkers(valContext.shoot.Spec.Provider.Workers, workersPath)...)
-	for i, workerConfig := range valContext.workerConfigs {
-		allErrors = append(allErrors, validation.ValidateWorkerConfig(workerConfig, workerConfigPath(i))...)
+	for i, workerConfigContext := range valContext.workerConfigsContexts {
+		allErrors = append(allErrors, validation.ValidateWorkerConfig(workerConfigContext.workerConfig, workerConfigContext.dataVolumes, workerConfigPath(i))...)
```

Comment:
You also need to pass the root volume to `ValidateWorkerConfig`, see above.

Reply:
https://github.com/gardener/gardener-extension-provider-kubevirt/pull/87/files#r505654157